apps/web-giddh/src/app/multi-currency-reports/profit-loss/profit-loss-report.component.ts

```
@@ -0,0 +1,315 @@
+
+    /**
+     * Initializes the component with report data from the store.
+     *
+     * @memberof ProfitLossReportComponent
+     */
+    public ngOnInit() {
+        this.componentStore.reportDataList$.pipe(takeUntil(this.destroyed$)).subscribe((p) => {
+            if (p) {
+                let data = cloneDeep(prepareProfitLossData(p)) as ProfitLossData;
+                let cogs;
+                if (data && data.incomeStatement && data.incomeStatement.costOfGoodsSold) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)